Dockerfile

```diff
@@ -30,7 +30,7 @@ WORKDIR /app/
 RUN /usr/bin/composer install
```

Comment:
@mrchrisadams do you want to keep this file around? With travis it's no longer needed.

Reply:
I'm in favour of getting rid of all the stuff we needed for cloudbuild, but not travis We have it in source history anyway with git, and it adds extra noise to the project.

Reply:
Ok done.